README_arne182.md

```diff
@@ -32,39 +32,36 @@ More info about how to install this fork can be found [here](https://medium.com/
-
-
-## Stock Lane Keeping Assist (LKA)
-Arne has worked on recreating the lane keeping assist system present in your car for openpilot. It works with cruise control not engaged, attempting to steer to keep you inside your lane when it detects you are departing it.
+# Todo
 
+- [ ] Auto Lane change from Boggyver
 
-## [Dynamic Following Distance Profile](https://github.com/ShaneSmiskol/openpilot/blob/dynamic-follow/README.md)
-(outdated: on 0.5.8, `dynamic-follow` branch only): Three following distance (TR) profiles are available to select; 0.9 seconds, 2.7 seconds, and a custom tuned dynamic follow profile. The first two behave as your stock cruise control system does. Dynamic follow aims to provide a more natural feeling drive, adjusting your distance from the lead car based on your speed, your relative velocity with the lead car, and your acceleration (or deceleration). If the system detects the lead car decelerating, your car should start to brake sooner than a hard-coded TR value. Same with accelerating.
+- [ ] Traffic light detection from Littlemountainman
 
+# Features
 
-## Slow Mode (SLO)
-For cars with longitudinal control down to 0 mph, you have the option to activate SLO mode which enables you to set your car's cruise control under your car's limit. For example, you could coast along at 15, 10, or even 5 mph.
-
-## Acceleration Profiles (GAS)
-You can select from three acceleration profiles with the GAS button. If your car accelerates too slowly for your liking, this will solve that. **Recently added**: dynamic acceleration profile for users with comma pedals. This should provide a smoother acceleration experience in stop and go traffic.
-
-## Select Vision Model (on 0.5.8, `dynamic-follow` branch only)
-You can select whether you would like to use the wiggly model or the normal vision model for path planning. Wiggly has more torque and can better guess the road curvature without lane lines, but it occasionally crashes or mispredicts the path.
-
-## EON and openpilot Stats
-With the on-screen UI, you can view stats about your EON such as its temperature, your grey panda's GPS accuracy, the lead car's relative velocity, its distance, and more.
-
-Warning from kegman: `WARNING: Do NOT depend on OP to stop the car in time if you are approaching an object which is not in motion in the same direction as your car. The radar will NOT detect the stationary object in time to slow your car enough to stop. If you are approaching a stopped vehicle you must disengage and brake as radars ignore objects that are not in motion.`
+- Braking: 
+    - by angle(carstate), 
+    - by predicted angle in 2.5s(laneplanner), 
+    - by model(commaai), 
+    - acceleration measured by steering angle, 
+    - by curvature (mapd), 
+    - by mapped sign(stop, yield, roundabouts, bump, hump, traffic light, speed sign, road attribute)
```

Comment:
you get these from openstreetmap, right? when do you download this? do you need simcard for this?

Reply:
This is openstreetmap data. Best is to teether for best performance. If the car is on it will start to download the data once you have an accuracy of 2m or less. Can be seen on display, therefore only works with grey and black panda.